### DISCUSSION TITLE
Add history of images that you have reacted to

### DISCUSSION THREAD

**Discussion by Nekos4Lyfe** [civitai/civitai#507] [Ideas]
Lots of cool images on the site but there is no way to keep track of the images you have reacted to. It would be nice to have a section in your profile over images that you liked.

**Comment by WolffM** [comment]
One Idea for this is to just update the API for images:
https://github.com/civitai/civitai/wiki/REST-API-Reference

[GET /api/v1/images](https://github.com/civitai/civitai/wiki/REST-API-Reference#get-apiv1images)
^Give this endpoint a filter condition for the user that has reacted to the image, can even split it by the type of reaction so you can organize things you want to save by reaction.

**Comment by JustMaier** [comment]
This has been implemented :)

[Check it out](https://civitai.com/user/@me/images?section=reactions)

![image](https://github.com/civitai/civitai/assets/607609/828cea43-2a50-4afc-b3a7-7c60e429d67f)
